core/Pos/Aeson/Crypto.hs

```diff
@@ -0,0 +1,21 @@
+module Pos.Aeson.Crypto
+       (
+       ) where
+
+import           Universum
+
+import           Crypto.Hash   (HashAlgorithm)
+import           Data.Aeson    (FromJSON (..), ToJSON (..))
+import           Formatting    (sformat)
+
+import           Pos.Crypto    (AbstractHash, PublicKey, fullPublicKeyF, hashHexF)
+import           Pos.Util.Util (parseJSONWithRead)
+
+instance ToJSON (AbstractHash algo a) where
+    toJSON = toJSON . sformat hashHexF
```

Comment:
why not `show` here (for implementations to be consistent)?

Reply:
This isn't my code, I just moved it.